apps/yarn.lock

```diff
@@ -6724,6 +6787,36 @@ osenv@0, osenv@^0.1.0, osenv@^0.1.3:
     os-homedir "^1.0.0"
     os-tmpdir "^1.0.0"
 
+pa11y-ci@^1.2.0:
```

Comment:
Some `yarn.lock` churn when upgrading a dependency is normal, but the introduction of `pa11y` entries makes me particularly suspicious here.